pages/common.css.wiki

```diff
@@ -1,3 +1,6 @@
+/* Fonts */
+@import url('https://fonts.googleapis.com/css2?family=Noto+Emoji:wght@300..700&family=Overpass:ital,wght@0,100..900;1,100..900&display=swap');
```

Comment:
Can we self-host the font? Using google fonts adds user tracking and is often not faster than re-using the same https connection.

Reply:
It's a bit more involved but we could have something that symlinks the font into the the directory that mediawiki serves static files from.